planetary-social/nos
# New gallery view

## Issues covered
#1177

## Description
Adds the new gallery view for posts with multiple links. You'll need to enable it in Settings under "Enable new media display" since we have lots of other things still to do:
- #1165 
- #1168 
- #1171 
- #1172 
- #1174 

(There are even more, but linking to all of them seems unnecessary.) The point is: this is just getting us started. We have a lot of work ahead of us. I'm not 100% sure if we should keep all of this behind the feature flag or if we should release some of it once we have it ready. I suppose that's a question for Linda.

## How to test
1. Navigate to Settings
2. Toggle "Enable new media display" to on
3. Find a post or two with multiple links
4. Swipe through the links to see the orange indicator move

## Screenshots/Video
https://github.com/user-attachments/assets/b5dabd1d-bb85-4856-bc34-f3aa6f47bd97
